lib/import-export-visitor.js

```diff
@@ -415,17 +414,11 @@ function getBlockBodyInfo(visitor, path) {
     // If parent is a Program, we may need to add a "use strict"
     // directive to enable const/let in Node 4.
     needToAddUseStrictDirective = visitor.enforceStrictMode;
-
     break;
 
   case "BlockStatement":
     body = parent.body;
-
```

Comment:
Because it's a block statement won't it always have a parent?
If not what test can we create to cause it to fail?